rhsm.go

```diff
@@ -229,14 +239,21 @@ func registerActivationKey(orgID string, activationKeys []string, serverURL stri
 		return err
 	}
 
+	options := make(map[string]string)
+	if len(environments) != 0 {
+		options["environment_names"] = strings.Join(environments, ",")
+		options["environment_type"] = environmentType
```

Comment:
I think that it is not necessary to pass `environmentType` as an argument to this function and you can use:

```go
options["environment_type"] = EnvTypeContentTemplate
```

because we will use only `"content-template"`.